src/components/form/adapters/widget-adapter.ts

```diff
@@ -109,20 +110,23 @@ export class LimeElementsWidgetAdapter extends React.Component {
             ...events,
         };
 
-        return React.createElement(LimeElementsAdapter, {
-            name: name,
-            elementProps: {
-                value: value,
-                label: this.getLabel(),
-                disabled: disabled,
-                readonly: readonly,
-                required: this.isRequired(),
-                invalid: this.isInvalid(),
-                'helper-text': this.getHelperText(),
-                ...extraProps,
-            },
-            events: newEvents,
-        });
+        return [
```

Comment:
It's better to render a fragment than an array in this case (avoids warning about missing `key`). Also, we could conditionally return just a single element when there's no help component.

Reply:
0aa08cf68e1905e1dba4b720ee48236b1a496369